docs/source/examples/text_examples.rst

```diff
@@ -0,0 +1,13 @@
+Text
+=================================
+
+.. manim:: Text1
+    :quality: medium
+    :save_last_frame:
+
+    class Text1(Scene):
+        def construct(self):
+            t = TextMobject("Hello World")
+            self.add(t)
+            self.wait(1)
```

Comment:
Comment (non-blocking): I've always found it annoying that we have to add a `wait` whenever we want to render a single frame. I've seen others do something like `self.play(Animation())`, but that doesn't seem to be any better. Perhaps we should open a new issue for manim to gracefully handle this situation.

Reply:
When you want to have only the last frame, you can also run this script:
```py
class Text1(Scene):
    def construct(self):
        t = TextMobject("Hello World")
        self.add(t)
```
and it will work with the -s flag.
Only when there should be a rendered video, there has to be at least one wait or play.
So for me, this is quit the behaviour I would expect.
Here, I only forgot to remove the self.wait, but I can also remove it now :)